gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/RecommendedProductDependenciesPlugin.groovy

```diff
@@ -0,0 +1,27 @@
+package com.palantir.gradle.dist
+
+import com.palantir.gradle.dist.tasks.CreateManifestTask
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.plugins.JavaPlugin
+import org.gradle.api.tasks.bundling.Jar
+
+class RecommendedProductDependenciesPlugin implements Plugin<Project> {
+
+    @Override
+    void apply(Project project) {
+        project.plugins.apply('java')
+        def ext = project.extensions.create('recommendedProductDependencies', RecommendedProductDependenciesExtension)
```

Comment:
nit: name is long, but accurate. if there was a just as accurate, shorter name, would prefer that